.circleci/config.yml

```diff
@@ -2,9 +2,11 @@ version: 2.1
 
 orbs:
  android: circleci/android@0.2.0
+ windows: circleci/windows@2.2.0
+
 
 jobs:
-  build:
+  build_APP:
     executor: android/android
```

Comment:
What's the difference between build and build_APP

Reply:
the difference is _APP